addons/godot-playfab/PlayFabClientConfig/PlayFabClientConfigLoader.gd

```diff
@@ -0,0 +1,55 @@
+extends Reference
+class_name PlayFabClientConfigLoader
+
+
+const DEBUG_DO_NOT_ENCRYPT = false	# Only works on debug builds
+const SECTION_NAME = "PlayFab"
+
+
+var _load_path = "user://playfab_client_config.cfg"
+var _config: ConfigFile = ConfigFile.new()
+var _title_id: String
+var _errors = []
+
+
+func save(title_id: String, new_config: PlayFabClientConfig):
+	_config.set_value(SECTION_NAME, "session_ticket", new_config.session_ticket)
```

Comment:
Can we simplify the set_value calls to one generic call and iterate over members?

Reply:
done